consoleapp.py

```diff
@@ -0,0 +1,27 @@
+from alonerpg.tablesreader import *
+
+def print_data(data):
+    if isinstance(data, str):
+        print(data)
+        return
+    if isinstance(data, list):
+        for table in data:
+            if isinstance(table, Table):
+                print(str(table.index) + ": (" + table.system + ") " + table.filename)
+    
+    print("")
+
+def main():
+    print("Bienvenido al programa de tablas aleatorias")
+    tables = get_tables()
+    print(ayuda().data)
```

Comment:
Function name in Spanish

Reply:
Not changing it. This is not only a function name, but also a command that the final user will type in the terminal, so it needs to be in spanish